helios.js

```diff
@@ -7002,6 +7002,14 @@ export class CborData {
 		}
 	}
 
+	/**
+	 *
+	 * @param {number[]} bytes
```

Comment:
I feel the input type should be `string` as that is usually the input type of a function called encodeUtf8 (eg. see plutus-core spec)

Reply:
Yeah that makes more sense! I'll have a play